celestia/peers.txt

```diff
@@ -8,3 +8,4 @@ f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056
 d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
 a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656
 2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756
+72dd61ecab64873e45c77f902fec73069dfb33f0@103.219.171.65:26656
```

Comment:
please submit a separate file in this director named lemniscap.json or simlply don't change this file

Reply:
this is to avoid any conflicts in the peers.txt file.

Reply:
:+1: will do.